### ISSUE TITLE
can't download apk that managed by pgyer.com

### ISSUE THREAD

**Issue by zoujinbin** [fluttercommunity/flutter_downloader#73]
The following is the code that download the apk which managed by pgyer.com, but it failed.
```
final taskId = await FlutterDownloader.enqueue(
        url: 'https://www.pgyer.com/apiv2/app/install?appKey=xxxx&_api_key=xxxxx',
        headers: {'content-type': 'application/x-www-form-urlencoded'},
        fileName: 'test.apk',
        savedDir: localPath.path,
        showNotification: false,
        openFileFromNotification: false);
```
The api is a http get request, and the `content-type` of the `Header` is `application/x-www-form-urlencoded`. How can I download the apk?

**Comment by zoujinbin** [comment]
The api doc : https://www.pgyer.com/doc/view/api#installApp

**Comment by zoujinbin** [comment]
![rsp](https://user-images.githubusercontent.com/39855077/58846857-4e27a600-86b3-11e9-9a5d-ca8526f6164d.jpg)
From the debug log, I saw it happened redirect. Is it the realson that I can't download the apk file. And What should I do?

**Comment by hnvn** [comment]
Look like a problem of URL redirect, this plugin has not handled this case in Android.

**Comment by hnvn** [comment]
fixed in v1.1.9
